src/components/Gallery/Gallery.js

```diff
@@ -0,0 +1,345 @@
+  '100%': {opacity: 1}
+})
+
+const swipeAnimationDurationMs = 200
+const swipeAnimation = (side = 'top') => {
+  const kfrms = css.keyframes({
+    '0%': {opacity: 1, [side]: 0},
+    '100%': {opacity: 0, [side]: 60}
+  })
+  return `${kfrms} ${swipeAnimationDurationMs}ms ease-out`
+}
+
+const styles = {
+  wrapper: css({
+    position: 'fixed',
+    top:0,
+    right: 0,
+    bottom:0,
+    left:0,  
+    zIndex: 100  
+  }),
+  gallery: css({
+    width: '100vw',
+    height: '100vh',
+    color: '#fff',
+    background: '#000',
+    display: 'flex',
+    flexDirection: 'column',
+    animation: `${fadeIn} ${fadeInDurationMs}ms ease-out`
+  }),
+  header: css({
+    flex: '0 0 auto',
+    display: 'flex',
+    justifyContent: 'space-between',
+    position: 'relative',
+    transition: 'opacity 0.1s ease-in',
+    padding: '30px 70px',
+    [onlyS]: {
+      padding: 15,
+    }
+    }),
+  body: css({
+    display: 'flex',
+    flex: '1 1 auto',
+  }),
+  counter: css({
+    flex: 1,
+  }),
+  close: css({
+    position: 'absolute',
+    flex: 1,
+    right: 68,
+    top: 28,
+    [onlyS]: {
+      right: 13,
+      top: 13,
+    }
+  }),
+  media: css({
+    height: '100px',
+  }),
+  caption: css({
+    flex: '0 0 auto',
+    padding: '15px 70px',
+    transition: 'opacity 0.1s ease-in',
+    [onlyS]: {
+      padding: 15,
+    }
+  }),
+  mediaItem: css({
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'row',
+    position: 'relative',
+    flex: 1,
+    flexGrow: 1,
+    opacity: 1
+  }),
+  mediaItemImage: css({
+    flex: '0 0 auto',
+    verticalAlign: 'bottom',
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    maxWidth: '100%',
+    maxHeight: '100%',
+    margin: 'auto',
+    animation: `${fadeIn} ${fadeInDurationMs}ms ease-out`,
+  }),
+  nav: css({
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    bottom: 0,
+    right: 0,
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: '100vw',
+    height: '100vh',
+    color: '#fff',
+    zIndex: zIndex.overlay,
+    '& .left': {
+      justifyContent: 'flex-start',
+    },
+    '& .right': {
+      justifyContent: 'flex-end'
+    }
+  }),
+  closing: css({
+    animation: swipeAnimation('top'),
+  }),
+  exitLeft: css({
+    animation: swipeAnimation('right'),
+  }),
+  exitRight: css({
+    animation: swipeAnimation('left'),
+  }),
+  navClose: css({
+    position: 'fixed',
+    width: 120,
+    height: 120,
+    top: 8,
+    right: 8
+  }),
+  navArea: css({
+    display: 'flex',
+    alignItems: 'center',
+    width: '33.333%',
+    height: '100vh',
+    padding: '0 30px',
+    opacity: 0,
+    transition: 'opacity 0.1s linear',
+    ':hover': {
+      transition: 'opacity 0.5s linear',
+      opacity: '0.7'
+    }
+  }),
+  navButton: css({
+    color: '#fff',
+    display: 'none',
+    [lUp]: {
+      display: 'block',
+    }
+  }),
+}
+
+function removeQuery(url) {
```

Comment:
Wearing the ES6 hat again:
`const removeQuery = url => (url || '').split('?')[0]`

Reply:
I agree that the ES6 syntax looks nicer – the reason I still use `function` sometimes (in reality, whenever I'm not writing an inline lambda) is that the name of a named function appears in stack traces whereas the name of the variable a lambda is bound to won't.

That said, I agree that we should agree on a style…